tests/core/contracts/test_contract_call_interface.py

```diff
@@ -184,6 +188,9 @@ def test_call_get_string_value(string_contract, call):
     assert result == "Caqalai"
 
 
+@pytest.mark.skipif(
+    LooseVersion(eth_abi.__version__) >= LooseVersion("2"),
+    reason="eth-abi >=2 does utf-8 string decoding")
 def test_call_read_string_variable(string_contract, call):
     result = call(contract=string_contract,
                   contract_function='constValue')
```

Comment:
Maybe add a second test that shows what happens on an ABI decoding error. We should probably do something like capture the eth-abi error and re-raise a generic `ValueError`.

Reply:
It raises a 'BadFunctionCallOutput' error